deploy/charts/firefly/templates/_helpers.tpl

```diff
@@ -124,6 +124,23 @@ blockchain:
       {{- if .Values.config.ethconnectPrefixLong }}
       prefixLong: {{ .Values.config.ethconnectPrefixLong }}
       {{- end }}
+{{- else if .Values.config.fabconnectUrl }}
+blockchain:
+  type: fabric
+  fabric:
+    fabconnect:
+      url: {{ tpl .Values.config.fabconnectUrl . }}
+      {{- if and .Values.config.fabconnectUsername .Values.config.fabconnectPassword }}
+      auth:
+        username: {{ .Values.config.fabconnectUsername | quote }}
+        password: {{ .Values.config.fabconnectPassword | quote }}
+      {{- end }}
+      retry:
+        enable: {{ .Values.config.fabconnectRetry }}
```

Comment:
Think this is a great thing to add, but we should add it consistently for `ethconnectRetry` too

Reply:
done, thanks!